packages/fury-adapter-remote/lib/adapter.js

```diff
@@ -0,0 +1,131 @@
+const deckardcain = require('deckardcain');
+const axios = require('axios');
+
+const name = 'remote';
+
+const outputMediaType = 'application/vnd.refract.parse-result+json; version=1.0';
+
+const defaultOptions = {
+
+  // the default value, but consumers should be able to override to use their own deployment
+  url: 'https://api.apielements.org',
+
+  parseEndpoint: '/parser',
+  validateEndpoint: '/validate',
+  serializeEndpoint: '/composer',
+
+  // the collection of "parse", media types we want this
+  // instance of the adapter to handle.
+  // NOTE, this allows you to use the API for one media type but
+  // another local adapter for another.
+
+  mediaTypes: [
+    'text/vnd.apiblueprint',
+    'application/swagger+json',
+    'application/swagger+yaml',
+
+    // serialization - there is noa actually mechanism to check supported format
+    // per individual operation in Fury
+    'application/vnd.refract+json',
+    'application/vnd.refract.parse-result+json',
+  ],
+
+  // fallback to try send input, if not indentified by deckardcain
+  defaultInputMediaType: 'text/vnd.apiblueprint',
+  defaultSerializationMediaType: 'application/vnd.refract.parse-result+json',
+};
+
+const detectMediaType = (source, defaultMediaType) => {
+  const mediaType = deckardcain.identify(source);
+  return mediaType || defaultMediaType;
+};
+
+class FuryRemoteAdapter {
+  constructor(options) {
+    this.name = name;
+    this.options = options || defaultOptions;
+    this.mediaTypes = this.options.mediaTypes;
+  }
+
+  detect(source) {
+    return this.mediaTypes.indexOf(deckardcain.identify(source)) !== -1;
```

Comment:
There's an [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method on array which can be used:

```suggestion
    return this.mediaTypes.includes(deckardcain.identify(source));
```